topics/single-cell/faqs/single_cell_omics.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Use our single cell Galaxy instance
+title: Use our Single Cell Lab
 area: Resources
 box_type: tip
 layout: faq
 contributors: [nomadscientist]
 ---
 
-Did you know we have a unique Galaxy instance with all our single cell tools highlighted to make it easier to use? We recommend this instance for all your single cell analysis needs, particularly for newer users.
+Did you know we have a unique Single Cell Lab with all our single cell tools highlighted to make it easier to use on Galaxy? We recommend this site for all your single cell analysis needs, particularly for newer users.
 
-This **subdomain** is uses the main European Galaxy infrastructure and power, it's just organised better for users of particular analyses...like single cell!
+The Single Cell Lab currently uses the main European Galaxy infrastructure and power, it's just organised better for users of particular analyses...like single cell!
 
-Try it out! All your histories/workflows/logins from the general [European Galaxy instance](https://usegalaxy.eu) will be there!
+Try it out! All your histories/workflows/logins from the general [European Galaxy server](https://usegalaxy.eu) will be there!
 
- - <i class="fa fa-tv" aria-hidden="true"></i> [Single cell omics Galaxy instance](https://singlecell.usegalaxy.eu)
+ - <i class="fa fa-tv" aria-hidden="true"></i> [Single Cell Lab](https://singlecell.usegalaxy.eu)
```

Comment:
could you add this to `_config.yml` as a named icon please?

Reply:
yes ma'am!